src/styles.css

```diff
@@ -308,3 +308,9 @@ a.dropdown-item:hover {
 .icon:hover {
   opacity: 0.8;
 }
+
+.config-label {
+  font-weight: 500 !important;
+  font-size: 0.9rem !important;
+  padding-top: 3px !important;
```

Comment:
2px seems good on ubuntu/ff. 3px is causing issue.

Also please check 
. Enabled label in last step of south/north etc wizard.
. Notifcations wizard and instances.